packages/geo/src/lib/query/shared/query.service.ts

```diff
@@ -42,6 +42,8 @@ import { MapExtent } from '../../map/shared/map.interface';
 })
 export class QueryService {
   public queryEnabled = true;
+  public workspace;
```

Comment:
I don't think that queryService should have workspace attribute. It should not know the existence of which workspace is open

Reply:
I agree my first idea was to had queryService IN Workspace, i think that is a better logic if the workspace have queryService in self. But the build libs doesn't work that way :(. the Workspace is in common lib and queryService in Geo and we have build problem that way. So i invert logic and had workspace in QueryService. 

the queryService.workspace is alway the good one i recheck that right now, garantie to the susbcribe in Workspace.state when it init: 
 https://github.com/infra-geo-ouverte/igo2-lib/pull/938/files#diff-d56f94d123e69f3aef4ee1cbe88a8c422b845daffff9d75bec683f7a85c6dd09R64

Reply:
The other way would be to add queryService in each workspace type (feature, wms, wfs) in geo ?